crates/topos-p2p/src/runtime/mod.rs

```diff
@@ -38,32 +27,20 @@ pub struct Runtime {
 
     /// Shutdown signal receiver from the client
     pub(crate) shutdown: mpsc::Receiver<oneshot::Sender<()>>,
+
+    pub(crate) current_bootstrap_id: Option<QueryId>,
```

Comment:
Could we get a doc comment here?

Reply:
the `current_bootstrap_id` will be use to track the different query of the current `bootstrap` of kademlia that we'll perform over time.

Reply:
@Freyskeyd Can you elaborate on that? I don't quite understand. And +1 for a doc comment here.